infrastructure/nomad/playbooks/templates/jobs/instant-bridge.nomad.j2

```diff
@@ -0,0 +1,191 @@
+
+    {% if env == 'devenv' %}
+    restart {
+      attempts = 0
+      mode = "fail"
+    }
+
+    reschedule {
+      attempts = 0
+      unlimited = false
+    }
+    {% endif %}
+
+    network {
+      mode = "bridge"
+
+      dns {
+        servers = {{ (ansible_facts['dns']['nameservers'] + ['1.1.1.1']) | tojson }}
+      }
+
+      {% for port_name, port_details in job.ports[0].items() %}
+      port "{{ port_name }}" {
+        {% if port_details.get('static') %}
+        static = {{ port_details['static'] }}
+        {% endif %}
+        {% if port_details.get('to') %}
+        to = {{ port_details['to'] }}
+        {% endif %}
+      }
+      {% endfor %}
+    }
+
+    {% for port_name in job.ports[0] %}
+    service {
+      name = "{{ job.name }}"
+      port = "{{ port_name }}"
+      tags = ["{{ port_name }}"]
+      provider = "nomad"
+      {% if port_name == "http" %}
+      check {
+        type     = "http"
+        path     = "/health"
+        interval = "10s"
+        timeout  = "2s"
+      }
+      {% endif %}
+    }
+    {% endfor %}
+
+    task "instantbridge" {
+      driver = "exec"
+
+      resources {
+        cpu = 4000
+        memory = 4096
+      }
+
+      artifact {
+        source = "https://foundry.paradigm.xyz"
+        destination = "local/foundry.sh"
+      }
+
+      {% if env != 'devenv' %}
+      artifact {
+        source = "https://primev-infrastructure-artifacts.s3.us-west-2.amazonaws.com/instant-bridge_{{ version }}_Linux_{{ target_system_architecture }}.tar.gz"
+      }
+      {% else %}
+      artifact {
+        source = "http://{{ ansible_facts['default_ipv4']['address'] }}:1111/instant-bridge_{{ version }}_Linux_{{ target_system_architecture }}.tar.gz"
+      }
+      {% endif %}
+
+      template {
+        data = <<-EOH
+          XDG_CONFIG_HOME="local/.config"
+          INSTANT_BRIDGE_LOG_LEVEL="{{ job.env.get('log-level', 'info') }}"
+          INSTANT_BRIDGE_LOG_FMT="{{ job.env.get('log-format', 'json') }}"
+          INSTANT_BRIDGE_LOG_TAGS="{{ 'service.name:' + job.name + '-{{ env "NOMAD_ALLOC_INDEX" }}' + ',service.version:' + version }}"
+          {%- raw %}
+          INSTANT_BRIDGE_KEYSTORE_DIR="/local/data-{{ env "NOMAD_ALLOC_INDEX" }}/keystore"
+          INSTANT_BRIDGE_KEYSTORE_FILENAME="{{ with secret "secret/data/mev-commit" }}{{ .Data.data.instant_bridge_keystore_filename }}{{ end }}"
+          INSTANT_BRIDGE_KEYSTORE_PASSWORD="{{ with secret "secret/data/mev-commit" }}{{ .Data.data.instant_bridge_keystore_password }}{{ end }}"
+          {{- range nomadService "mev-commit-geth-bootnode1" }}
+            {{- if contains "http" .Tags }}
+          INSTANT_BRIDGE_SETTLEMENT_RPC_URL="http://{{ .Address }}:{{ .Port }}"
+            {{- end }}
+          {{- end }}
+          {{- range nomadService "{% endraw %}{{ job.target.name }}{% raw %}" }}
+            {{- if contains "rpc" .Tags }}
+          INSTANT_BRIDGE_BIDDER_RPC_URL="{{ .Address }}:{{ .Port }}"
```

Comment:
Does the instant bridge's bidder node run within the instant bridge process or is it a separate process?

Reply:
Its a separate process. We configure the target here. So the IP/Port of the running bidder node configured. It is configured in profiles.yml as 'target'.